scripts/yosys/synth.tcl

```diff
@@ -343,15 +345,15 @@ design -save checkpoint
 # Explore/Finalize
 if { [info exists ::env(SYNTH_EXPLORE)] && $::env(SYNTH_EXPLORE) } {
     for { set index 0 }  { $index < [llength $delay_scripts] }  { incr index } {
-        set name "DELAY$index"
+        set name "DELAY $index"
```

Comment:
wont the output netlist contain a space now ?

Reply:
You were right. Resolved.